platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/hashmap/Bucket.java

```diff
@@ -251,7 +232,8 @@ public void putValue(final int keyHashCode, final K key, final long value) {
                         // move all entries after this one up
                         final int offsetOfNextEntry = result.entryOffset + entrySize;
                         final int sizeOfEntriesToMove = currentSize - offsetOfNextEntry;
-                        //  FUTURE WORK For Java 17 do this https://github.com/swirlds/swirlds-platform/issues/4090
+                        //  FUTURE WORK For Java 17 do this
```

Comment:
I think so. System.arraycopy() is still used instead of the new ByteBuffer API. Since there is a separate task about it, I'd not include it to this PR as unrelated.